apps/address-service/domains/common/utils/services/search/searchServiceUtils.js

```diff
@@ -84,4 +94,48 @@ async function createReturnObject ({
     return ret
 }
 
-module.exports = { createOrUpdateAddressWithSource, createReturnObject }
+function sortObject (obj) {
+    if (typeof obj !== 'object' || obj === null) {
+        return obj
```

Comment:
when you use single return in condition, no curly braces required, you can write it in one line of code